src/index.css

```
@@ -0,0 +1,648 @@
+  --color-info-dark: var(--info-dark);
+  --color-info-foreground: var(--info-foreground);
+
+  --animate-shine: shine var(--duration) infinite linear;
+
+  --hero-bg: var(--background);
+  --hero-fg: var(--foreground);
+  --hero-primary: var(--primary);
+  --hero-accent: var(--secondary);
+  --hero-muted: var(--muted);
+
+  --animate-shine: shine var(--duration) infinite linear;
```

Stylelint (17.7.0)

[error] Duplicate custom property "--animate-shine" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)